torch/fx/node.py

```diff
@@ -628,6 +628,15 @@ def maybe_replace_node(n : Node) -> Node:
         assert isinstance(new_kwargs, dict)
         self.__update_args_kwargs(new_args, new_kwargs)
 
+    def rename(self, candidate: str):
```

Comment:
meh, let's not add this as a public API to FX, can you keep this somewhere else? (If you do want to propose this as API, need one of those `@compitability` decorators

Reply:
or maybe just underscore it